DEVELOPING.md

```diff
@@ -125,7 +125,7 @@ load_package /hab/cache/artifacts/core-elasticsearch*.hart
 
 *NOTE*: By default, an automated fetch from the https://bldr.habitat.sh upstream is configured so that 'core' and 'habitat' origin packages are automatically retrieved from the upstream. Make sure you have those origins created. The upstream fetch happens in the background. To trigger a fetch for a non-existent package, you can do a local 'hab pkg search' for that package - it will initially fail with a 'Not Found', but will trigger a background fetch. If you run into any issues, check the upstream log file in the '/hab/svc/builder-api/var' directory.
```

Comment:
Not that this is part of your PR, but this section on the upstream isn't correct - we don't have the upstream automatically configured, as it can cause subtle changes that become confusing when deving.  So it needs to be explicitly turned on if you want/need it.